README.md

```diff
@@ -0,0 +1,257 @@
+
+## Rake Tasks
+
+The system provides various Rake tasks for data collection:
+
+```bash
+# ENTSOE data collection
+rake entsoe:all              # Run all ENTSOE tasks
+rake entsoe:generation       # Collect generation data
+rake entsoe:load             # Collect load data
+rake entsoe:price            # Collect price data
+rake entsoe:transmission     # Collect transmission data
+
+# ELEXON data collection
+rake elexon:all              # Run all ELEXON tasks
+rake elexon:fuelinst         # Collect fuel mix data
+rake elexon:load             # Collect load data
+rake elexon:unit             # Collect unit data
+
+# Other market operators
+rake aemo:all                # Run all AEMO tasks
+rake nordpool:all            # Run all Nordpool tasks
+```
+
+## Database Schema
+
+The system uses a TimescaleDB-powered PostgreSQL database with the following structure:
+
+```mermaid
+erDiagram
+    Areas {
+        smallint id PK
+        string code
+        string name
+        string source
+        string internal_id
+        string type
+        string region
+        string electricitymaps_id
+    }
+    ProductionTypes {
+        smallint id PK
+        string name
+        string name2
+        boolean enabled
+    }
+    Units {
+        int id PK
+        int area_id FK
+        int production_type_id FK
+        string internal_id
+        string code
+    }
+    GenerationUnits {
+        int unit_id PK,FK
+        timestamp time PK
+        int value
+    }
+    Generation {
+        int area_id PK,FK
+        int production_type_id PK,FK
+        timestamp time PK
+        int value
+    }
+    Load {
+        int area_id PK,FK
+        timestamp time PK
+        int value
+    }
+    Transmission {
+        int areas_area_id FK
+        timestamp time PK
+        int value
+    }
+    AreasAreas {
+        smallint id PK
+        smallint from_area_id FK
+        smallint to_area_id FK
+    }
+    Prices {
+        int area_id PK,FK
+        timestamp time PK
+        decimal value
+    }
+    DataFiles {
+        string path PK
+        string source PK
+        timestamp updated_at
+    }
+
+    Areas ||--o{ Units : "has"
+    Areas ||--o{ Generation : "has"
+    Areas ||--o{ Load : "has"
+    Areas ||--o{ Prices : "has"
+    Areas ||--o{ AreasAreas : "from"
+    Areas ||--o{ AreasAreas : "to"
+    ProductionTypes ||--o{ Units : "type"
+    ProductionTypes ||--o{ Generation : "type"
+    Units ||--o{ GenerationUnits : "generates"
+    AreasAreas ||--o{ Transmission : "connects"
+```
+
+Key features of the schema:
+
+1. **Time-Series Tables**
+   - `generation`, `load`, `transmission`, `prices` - Core time-series data
+   - Optimized with TimescaleDB hypertables
+   - Compressed for efficient storage
+
+2. **Reference Tables**
+   - `areas` - Geographic/market regions
+   - `production_types` - Types of power generation
+   - `units` - Individual generation units
+   - `areas_areas` - Area interconnections
```

Comment:
areas_areas and areas_production_types are join tables (has_and_belongs_to_many), maybe should be separate section.

Previously transmission has from_area_id and to_area_id columns and generation had area_id and production_type_id columns. Moving these to separate tables reduces table size, improves indexing, simplifies query plan etc.